Document usage of `./gradlew format` and workaround for files without import statements (#3269)
airbytehq/airbyte: docs/contributing-to-airbyte/developing-locally.md

./gradlew :airbyte-tests:acceptanceTests
```

## Run formatting automation/tests

> To run our formatting scripts, simply run `./gradlew format` at the base of the repo. 

Reviewer: just less emphasis on "our" 
```suggestion
To format code in the repo , simply run `./gradlew format` at the base of the repo. 
```

Author: Done!
